.github/workflows/squash.yml

```
@@ -0,0 +1,39 @@
+name: Squash vuln-list repo
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_OWNER: ${{ github.repository_owner }}
+      GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+    steps:
+    - name: Install dependencies
+      run: |
+        sudo apt-get -y update
+        sudo apt-get -y install git-filter-repo
+
+    - name: Setup github user email and name
+      run: |
+        git config --global user.email "action@github.com"
+        git config --global user.name "GitHub Action"
+
+    - name: clone a shallow repo
+      uses: actions/checkout@v3
+      with:
+        repository: ${{ github.repository_owner }}/vuln-list
+        token: ${{ secrets.ACCESS_TOKEN }}
+        path: vuln-list
+        fetch-depth: 100
+
+    - name: squash and push
+      run: |
+        cd vuln-list
+        git replace -f --graft $(git rev-list --max-parents=0 HEAD)
+        git filter-repo --force
+        git remote add origin https://github.com/afdesk/vuln-list.git
+        git push --force --set-upstream https://${GITHUB_TOKEN}@github.com/afdesk/vuln-list.git main
```

We have to use `ACCESS_TOKEN`, not `GITHUB_TOKEN`, right?

```suggestion
        git push --force --set-upstream https://${ACCESS_TOKEN}@github.com/aquasecurity/vuln-list.git main
```

---

Using `github.repository_owner` may be better.

```suggestion
        git push --force --set-upstream https://${GITHUB_TOKEN}@github.com/${{ github.repository_owner }}/vuln-list.git main
```